Mark9Mbugua/SendITAPI
# #161729000 Create a folder structure

**What does this PR do?**
It creates a project folder structure

**Description of Task to be completed?**
Create the skeleton structure of the project and organize all the folders and files accordingly

 **How should this be manually tested?**
See this repo or clone it into a local repo and navigate through all the folders

 **Any background context you want to provide?**
I have created the README.md and .gitignore files.

 **What are the relevant pivotal tracker stories?**
#161729000
